editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/util/maven/MavenUtils.java

```diff
@@ -73,14 +81,45 @@ public static void updateContributedPlugins(List<Plugin> plugins, String camelVe
 	 * @param dependencies
 	 * @param camelVersion
 	 */
-	public static void updateCamelVersionDependencies(List<Dependency> dependencies, String camelVersion) {
-		for (Dependency dep : dependencies) {
-			if ("org.apache.camel".equalsIgnoreCase(dep.getGroupId()) && dep.getArtifactId().startsWith("camel-")) {
-				dep.setVersion(camelVersion);
+	public static void updateCamelVersionDependencies(Model mavenModel, List<Dependency> dependencies, String camelVersion) {
+		Properties properties = mavenModel.getProperties();
+		if(isMavenPropertyCamelVersionSet(properties)){
+			properties.setProperty(MAVEN_PROPERTY_CAMEL_VERSION, camelVersion);
+		}
+		if(camelVersion.contains("redhat")){
```

Comment:
Should we make "redhat" into a constant somewhere so that we can more easily change it if we need to in the future?